custom_components/telenet_telemeter/switch.py

```
@@ -10,7 +10,8 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=15)
+MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=240)
```

Ruff (0.8.2)

`timedelta` may be undefined, or defined from star imports

(F405)